Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -191,8 +191,9 @@ private void OnListen(EntityUid uid, RadioMicrophoneComponent component, ListenE
         if (HasComp<RadioSpeakerComponent>(args.Source))
             return; // no feedback loops please.
 
-        if (_recentlySent.Add((args.Message, args.Source)))
-            _radio.SendRadioMessage(args.Source, args.Message, _protoMan.Index<RadioChannelPrototype>(component.BroadcastChannel), uid);
+        var channel = _protoMan.Index<RadioChannelPrototype>(component.BroadcastChannel)!;
+        if (_recentlySent.Add((args.Message, args.Source, channel)))
+            _radio.SendRadioMessage(args.Source, args.Message, channel, uid);
```

Comment:
This is probably liable to be an issue if devices can ever be sent to multiple channels at once.
But I guess in that case perhaps it'd just be a matter of iterating over the channels, checking and sending to each individually?

Reply:
This isn't an issue, if radios were ever given this ability, there would have to be extensive changes made to this system in any case